packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -83,26 +106,36 @@ class VerticalPage extends React.Component {
 import React from 'react';
 import {
   Page,
-  PageHeader,
-  PageHeaderTools,
-  PageSidebar,
+  Masthead,
+  MastheadMain,
+  MastheadBrand,
+  MastheadContent,
   PageSection,
-  PageSectionVariants
+  PageSectionVariants,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem
 } from '@patternfly/react-core';
 
 HorizontalPage = () => {
-  const logoProps = {
-    href: 'https://patternfly.org',
-    onClick: () => console.log('clicked logo'),
-    target: '_blank'
-  };
+  const headerToolbar = (
+    <Toolbar id="toolbar">
+      <ToolbarContent>
+        <ToolbarItem>Navigation</ToolbarItem>
+        <ToolbarItem>header-tools</ToolbarItem>
+      </ToolbarContent>
+    </Toolbar>
+  );
+
   const Header = (
-    <PageHeader
-      logo="Logo"
-      logoProps={logoProps}
-      headerTools={<PageHeaderTools>header-tools</PageHeaderTools>}
-      topNav="Navigation"
-    />
+    <Masthead display={{ default: 'stack' }} inset={{ default: 'insetXs' }}>
```

Comment:
I don't think the forced stack display is needed here. The horizontal page indicates the navigation is inside the toolbar rather than in a separate sidebar.